habitat/plan.sh

```diff
@@ -64,40 +58,46 @@ do_prepare() {
   build_line "Ruby ABI version appears to be ${RUBY_ABI_VERSION}"
 
   build_line "Setting link for /usr/bin/env to 'coreutils'"
-  [[ ! -f /usr/bin/env ]] && ln -s $(pkg_path_for coreutils)/bin/env /usr/bin/env
-
-  return 0
+  [ ! -f /usr/bin/env ] && ln -s "$(pkg_path_for coreutils)/bin/env" /usr/bin/env || return 0
 }
 
 do_build() {
-  cd $CACHE_PATH
```

Comment:
Same, avoid changing directories like this. Use sub-shells instead. See line `87`